### DISCUSSION TITLE
Having issue with latex command `\not` and `\neq`

### DISCUSSION THREAD

**Discussion by Mr-Sheep** [standardnotes/forum#2898] [General]
**Describe the bug**

`a \not = b` and `a \neq b` are not shown correctly in `markdown alternative`

**To Reproduce**
1. use latex command `a \not = b` or `a \neq b` in `markdown alternative`

**Expected behavior**
expected output from `a \not = b` or `a \neq b`: $a \not = b$, and $a \neq b$

**Desktop (please complete the following information):**

MacOS Version 3.23.292

**Smartphone (please complete the following information):**
iOS v3.45.4 (288)

**Screenshots**
For Desktop: 
![CleanShot 2022-11-16 at 15 16 15@2x cleaned](https://user-images.githubusercontent.com/24599209/202296635-7db80c7d-111f-4f86-a8ef-87adc81bef71.png)

<img width="381" alt="CleanShot 2022-11-16 at 15 19 22@2x" src="https://user-images.githubusercontent.com/24599209/202296700-aa975c87-2f56-49e8-98ab-667b320dfc94.png">

For Mobile Client:
![IMG_71382E17B43D-1 cleaned](https://user-images.githubusercontent.com/24599209/202297084-8a106635-5e91-4c7d-8923-373d685bd930.jpeg)

**Comment by effieeee** [comment]
Can confirm, perhaps we need to look into an issue on our copy of the plugin for LaTeX
